orchestrator/api/api_v1/endpoints/processes.py

```diff
@@ -370,3 +373,34 @@ def processes_filterable(
         raise CacheHit(HTTPStatus.NOT_MODIFIED, headers=dict(response.headers))
 
     return [asdict(enrich_process(p)) for p in results]
+
+
+@router.websocket("/test/{pid}")
```

Comment:
is this for a POC?

Reply:
no, it had `/test/` because I was first testing it out and hadn't thought about changing it